tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -122,13 +122,13 @@ export class WebGPUBackend extends KernelBackend {
   private downloadWaitMs = 0;
   private dummyCanvas: HTMLCanvasElement;
   private dummyContext: GPUCanvasContext;
+  private gpuResourceDisposalQueue: DataId[] = [];
```

Comment:
Here we can use `tensorDataDisposalQueue`?

Reply:
Done. Please take another look.